src/shapes/IText/IText.ts

```diff
@@ -668,9 +676,9 @@ export class IText<
    * Unused by the library, is for the end user
    * @return {String | TFiller} Character color (fill)
    */
-  getCurrentCharColor(): string | TFiller {
+  getCurrentCharColor(): string | TFiller | null {
```

Comment:
fill and stroke being null is the cause of many issue with types and a lot of extra null checks.

Reply:
if we removing null why this this needed?

Reply:
because i m not removing null in this PR.
But this PR brought up the issue again